docs/web/10.offline-storage/service-workers.md

```diff
@@ -29,7 +29,9 @@ Service Worker 就这样诞生了，它在 Web Worker 的基础上加上了持
 5. 异步实现：内部大都是通过 Promise 实现 
 
 ## Service Worker 生命周期
-Service Worker 的生命周期完全独立于网页。
+Service Worker 的生命周期完全独立于网页。以下是 Service Worker 简化生命周期。
+
+![service worker lifecycle](https://cosmos-x.oss-cn-hangzhou.aliyuncs.com/sw_lifecycle.png)
```

Comment:
import Img component at the top of main content `import Img from '../../../src/components/Img'`

use Img component `<Img w="" src="">` and you can also adjust the size with `w` prop

![image](https://user-images.githubusercontent.com/12554487/70596231-517e8a00-1c20-11ea-8b8e-3ff58b9de9fa.png)